docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -40,25 +38,26 @@ StatefulSets are intended to be used with stateful applications and distributed
 systems. However, the administration of stateful applications and 
 distributed systems on Kubernetes is a broad, complex topic. In order to 
 demonstrate the basic features of a StatefulSet, and to not conflate the former 
-topic with the latter, you will deploy a simple web application using StatefulSets.
+topic with the latter, you will deploy a simple web application using a StatefulSet.
 
 After this tutorial, you will be familiar with the following.
 
 * How to create a StatefulSet
 * How a StatefulSet manages its Pods
 * How to delete a StatefulSet
 * How to scale a StatefulSet
-* How to update the container image of a StatefulSet's Pods
+* How to update a StatefulSet's Pods
+* How to use Pod Management policies 
```

Comment:
Pod Management policies is subsumed under "How a StatefulSet manages its Pods".  No need to mention it.  It already sounds way more important than it is.